src/components/MenuPageContainer/MenuPage.jsx

```diff
@@ -0,0 +1,136 @@
+import { useState } from 'react';
+import Header from '../HeaderContainer/Header';
+import Footer from '../FooterContainer/Footer';
+import Section from '../SectionContainer/Section';
+import Cart from '../CartContainer/Cart';
+import {
+  phoneTooltip,
+  containerMenu,
+  phoneTooltipHover,
+  menuPageButtons,
+  menuPageFilterButtons,
+  menuDescribe,
+  hMenu,
+  pMenu,
+  menuPageFilterButtonsContainer,
+} from './styles';
+import Button from '../ButtonComponent/Button';
+
+const MenuPage = () => {
+  const [selectedSection, setSelectedSection] = useState('Home');
```

Comment:
As far as you use this hardcoded word several times in your app, it would be better to extract it to a constant.